.github/workflows/runchecks.yml

```diff
@@ -6,6 +6,7 @@ jobs:
   build:
     env:
       WANT_ETHERNET_MAB: True
+      WANT_FDO: True
```

Comment:
Can we avoid this somehow? This requires that we run the test as: 
```bash
WANT_ETHERNET_MAB=1 WANT_FDO=1 pytest
```
and the tests fail if I run pytest without the env variables.

Reply:
I can only think of ways that involve a larger refactor. I'll create an issue and resolve this for now.

Reply:
Created #67